src/Tests/Microsoft.Diagnostics.Monitoring.Tool.UnitTests/ExceptionsPipelineTests.cs

```diff
@@ -110,7 +110,7 @@ public Task EventExceptionsPipeline_AsyncException()
                     Assert.Equal(typeof(TaskCanceledException).FullName, instance.TypeName);
                     Assert.False(string.IsNullOrEmpty(instance.Message));
 
-                    ValidateStack(instance, "ThrowForNonSuccess", "System.Private.CoreLib.dll", "System.Runtime.CompilerServices.TaskAwaiter");
+                    ValidateStack(instance, "ThrowForNonSuccess(System.Threading.Tasks.Task)", "System.Private.CoreLib.dll", "System.Runtime.CompilerServices.TaskAwaiter");
```

Comment:
While this is technically correct, we will probably want to simplify this to just be the simple name e.g. `Task` instead of `System.Threading.Tasks.Task`; the fully qualified name is going to cause too much verbosity. I would save this for a future change.

Reply:
/todo [Exceptions] Simplify Representation of Method Parameter Types